drafts/pair-programming-mistakes.md

```diff
@@ -0,0 +1,120 @@
+The driver does not switch with the navigator after a committable segment, and we continue to push forward.
+
+Contrary to popular belief, [being in the flow isn’t always advantageous](https://en.wikipedia.org/wiki/Flow%5F(psychology)#:~:text=In%20some%20cases,on%20attentional%20abilities.), since it can lead to fatigue and overworking.
+I noticed that when we are tired, the quality and quantity of our communication rapidly decreases.
+We all start to think about our own problem and ignore our partner, or we are stuck on the wrong problem, a phenomenon known as the [XY problem](https://en.wikipedia.org/wiki/XY%5Fproblem).
+This significantly reduces our productivity and may even render us ineffective.
+
+![Alt text (by mom)](https://cdn.hashnode.com/res/hashnode/image/upload/v1717162596435/FRNbO6MkS.png?auto=format)
+
+The key in avoiding this trap is to remember to take regular breaks, switch roles and ensure effective communication at all times. 
+This not only helps to keep our minds fresh but also ensures that we are on the same page throughout the whole session.
+
+
+## Remote Pairing under Non-Ideal Conditions
+There was an instance when a colleague and I attempted to pair program remotely. 
+We experienced initially minor, but with time major delay, which caused us frustration and irritation. 
+However, not wanting to constantly complain, we tried to cope with the situation.
+
+![Alt text (by mom)](https://cdn.hashnode.com/res/hashnode/image/upload/v1717162575632/jl%5F2eYy7A.png?auto=format)
+
+Unfortunately, the delay made it nearly impossible to follow the driver’s actions, and we eventually had to stop the pairing session. 
+This experience underscored the importance of having an excellent setup and high-quality connection for remote pair programming. 
+Even minor issues can become major distractions, diverting focus from the task at hand.
+
+If given the choice, it’s preferable to pair program in front of the same screen. 
+But if remote pairing is necessary, ensuring a low-latency high-quality connection is paramount to a successful and productive session.
+
+
+## Concluding Thoughts
+The effectiveness of pair programming is tremendously influenced by soft skills and the dynamics between the two individuals involved. 
+Factors such as personality traits (introvert vs extrovert), experience levels (junior vs senior), and familiarity with 
+each other (new to the team vs well-acquainted) can have a monumental impact on the quality and productivity of the pairing session.
+
+It’s essential to adjust the focus of the pairing session depending on the circumstances. 
+For instance, if the pair programmers are not well-acquainted with each other, the initial focus should be on establishing a good dynamic and trustworthy relationship.
+Furthermore, it’s crucial not to overwhelm a (junior) developer with too much information at once, whether it’s project specifics, domain knowledge, or IDE shortcuts. 
```

Comment:
Just a thought: maybe provide these points as a list --> "takeaway" points

Reply:
Good point! i think a list would work as well. My gut feeling tells me to prefer the text though, dono why..